README.md

```diff
@@ -1,2 +1,184 @@
 # trampss-redux-data-store
+
+Factory of Redux reducers and their associated actions and selectors.
+> Make your Redux code base tinier and simpler to maintain
+
 [![CircleCI](https://circleci.com/gh/Trampss/trampss-redux-data-store.svg?style=shield)](https://circleci.com/gh/Trampss/trampss-redux-data-store) [![Coverage Status](https://coveralls.io/repos/github/Trampss/trampss-redux-data-store/badge.svg?branch=master)](https://coveralls.io/github/Trampss/trampss-redux-data-store?branch=master) [![NPM Version](https://badge.fury.io/js/trampss-redux-data-store.svg)](https://www.npmjs.com/package/trampss-redux-data-store)
+
+## Contents
+ - [Purpose](#purpose)
+ - [Why](#why)
+ - [Installation](#installation)
+ - [API](#api)
+
+## Purpose
+`trampss-redux-data-store` generates generic reducers, actions and selectors in two lines.
+
+```es6
+import factory from 'trampss-redux-data-store'
+export default factory('id')('api')('todos')
+```
+That's it, you exported a function wich is a reducer to include in a Redux `combineReducer`.
```

Comment:
typo in which.

You exported a reducer function you can register thanks to `combinerReducer` in Redux